debian/control

```diff
@@ -38,6 +38,21 @@ Depends:
 Description: Updater for Endless OS
  This package contains the components for keeping Endless OS up to date.
 
+Package: eos-updater-tests
+Section: misc
+Architecture: any
+Depends:
+ eos-updater (= ${binary:Version}),
+ gir1.2-glib-2.0,
+ python3-gi,
+ ${misc:Depends},
+ ${python3:Depends},
+ ${shlibs:Depends},
+Description: Updater for Endless OS - tests
+ This package contains the components for keeping Endless OS up to date.
```

Comment:
You mean the repeated “This package contains the components for keeping Endless OS up to date.” line, or do you mean all the `Package` sections beneath there?

The "This package…” line is there because I thought it was the convention to repeat the description of the ‘main’ package in all the `-tools`, `-dev`, etc. packages for it, plus a suffix saying something specific about the development stuff which that package provides.

I think the fix here, then, is to reword the “This package…” line everywhere to be a bit more descriptive. Something like “Updater for Endless OS which runs periodically to download and apply updates to the core operating system. It includes system integration to automatically run and check for updates periodically, and facilities for distributing updates to peers on the local network.”?